specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json

```diff
@@ -2136,6 +2143,28 @@
         "value"
       ]
     },
+    "SSISExecutionCredential": {
+      "description": "SSIS package execution credential.",
+      "properties": {
+        "domain": {
+          "type": "object",

```

Comment:
Should this be string?

Reply:
This should be object, as we need support string or expression.